examples/src/main/java/io/kubernetes/client/examples/Example.java

```diff
@@ -9,36 +9,230 @@
+import io.kubernetes.client.models.ExtensionsV1beta1DeploymentSpec;
+import io.kubernetes.client.models.V1NamespaceList;
 import io.kubernetes.client.models.V1PodList;
+import io.kubernetes.client.models.V1ServiceList;
 import io.kubernetes.client.util.Config;
 import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
 
 /**
  * A simple example of how to use the Java API
  *
- * <p>Easiest way to run this: mvn exec:java
+ * <p>
+ * Easiest way to run this: mvn exec:java
  * -Dexec.mainClass="io.kubernetes.client.examples.Example"
  *
- * <p>From inside $REPO_DIR/examples
+ * <p>
+ * From inside $REPO_DIR/examples
  */
 public class Example {
-  public static void main(String[] args) throws IOException, ApiException {
-    ApiClient client = Config.defaultClient();
-    Configuration.setDefaultApiClient(client);
 
-    CoreV1Api api = new CoreV1Api();
-    V1PodList list =
-        api.listPodForAllNamespaces(null, null, null, null, null, null, null, null, null);
-    for (V1Pod item : list.getItems()) {
-      System.out.println(item.getMetadata().getName());
+    private final CoreV1Api corev1Api;
+    private final static String DEFAULT_NAME_SPACE = "default";
+    private final static Integer TIME_OUT_VALUE = 180;
+
+    /*
+    For API_SERVER_NAME, you can get the server name as follows.
+    $ kubectl cluster-info|grep master
+    Kubernetes master is running at https://*****************.hcp.japaneast.azmk8s.io:443
+     */
+    private final static String API_SERVER_NAME = "https://*****************.hcp.japaneast.azmk8s.io";
+    /*
+    For ACCESS_TOKEN , you can get the token as follows
+    $ kubectl describe secret $(kubectl get secrets | grep default | cut -f1 -d ' ') | grep -E '^token' | cut -f2 -d':' | tr -d '\t'
+     */
+    private static final String ACCESS_TOKEN = "********************************";
+    
+    private final static Logger LOGGER = Logger.getLogger(Example.class.getName());
+
+    /**
+     * Constructor
+     * @throws java.io.IOException
+     */
+    public Example() throws IOException {
+        ApiClient client = Config.defaultClient();
+        //If you want to use specific k8s cluster and access token, please use following?
+        //ApiClient client = Config.fromToken(API_SERVER_NAME, ACCESS_TOKEN, false);
+        Configuration.setDefaultApiClient(client);
+        corev1Api = new CoreV1Api(client);
+    }
+
+    /**
+     * Main method
+     *
+     * @param args
+     */
+    public static void main(String[] args) {
+        try {
+            Example operation = new Example();
+            operation.executeCommand();
+        } catch (ApiException | IOException ex) {
+            Logger.getLogger(Example.class.getName()).log(Level.SEVERE, null, ex);
+        }
+    }
+
+    public void executeCommand() throws ApiException {
+
```

Comment:
delete newline.

Reply:
If the above is OK, I will copy the code to main() method.